### PR TITLE
Better handling of specific cases and lint

### PR DESCRIPTION
**Pull request by Pierlou**
- `>` in enumerations is now correctly displayed
- inner links work better (were broken for fields with a dot)
- got rid of most `.format()` for clarity
- lint code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ venv
 build/
 dist/
 table_schema_to_markdown.egg-info/
+work/
```
Comment (maudetes) [inline .gitignore:6]: (You may add it to your own personal `.git/info/exclude` if it does not apply to other users)

File: table_schema_to_markdown/__init__.py [hunk 2]
```diff
@@ -225,71 +244,90 @@ def convert_json(schema_json, out_fd,style):
...
                 sizes = ""
                 pattern = ""
 
-                if field_constraints:
-                    required = None
-                    if field_constraints.get("required"):
-                        required = "Oui"
-                    elif not field_constraints.get("required", True):
-                        required = "Non"
+                if field_constraints and field_constraints.get("required"):
```
Comment (maudetes) [inline table_schema_to_markdown/__init__.py:313]: No need for a None case?

**Review by maudetes — APPROVED** [review]
Long live f-strings! :crown:

(Are we on python >= 3.6?)

File: table_schema_to_markdown/__init__.py (hunk 2 above)
Comment (Pierlou) [reply table_schema_to_markdown/__init__.py:313]: I don't think so, fields are either required or not, not specifying if they are means they are not (at least it's what we usually do)

**Comment by Pierlou** [comment]
Yes, the production environment is more recent than that (we only use this package there for now). Thanks for the quick review!

**Pull request merged**